src/components/tables/DataTable/pagination/Pagination.scss

```diff
@@ -0,0 +1,57 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@use '../../../../styling/defs.scss' as bk;
+
+
+.pagination {
```

Comment:
Please apply the usual boilerplate CSS here:  `@layer baklava.components`, class names with `bk-` prefix, the `@include bk.component-base(...)`.